masks/mask_class/mask_class.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import cv2 as cv
+from math import sqrt
+from sympy.ntheory import isprime, quadratic_residues
+from scipy.signal import max_len_seq
+from perlin_numpy import generate_perlin_noise_2d
+
+
+class Mask():
+
+    def __init__(self, sensor_size_px: tuple, 
+                 sensor_size_m: tuple, 
+                 feature_size: tuple, 
+                 distance_sensor: float) -> None:
+        
+        self.mask = None
+        self.sensor_size_px = sensor_size_px
+        self.sensor_size_m = sensor_size_m
+        self.feature_size = feature_size
+        self.distance_sensor = distance_sensor
+    
+    def shape(self):
+        return self.mask.shape
+
```

Comment:
an abstract `create_mask` method could be nice? So the process for each constructor would be:

1. Store mask-specific parameters
2. Call parent `Mask` class
    - Store common mask parameters
    - call `self.create_mask()` (which is defined for each child class)
    - call `self.compute_psf()`, for computing the PSF given the previously computed mask and the sensor distance